objects/rct2/music/rct2.music.pirate.json

```diff
@@ -0,0 +1,42 @@
+{
+    "id": "rct2.music.pirate",
+    "authors": ["Chris Sawyer", "Allister Brimble"],
+    "version": "1.0",
+    "originalId": "0000008E|#MUSIC30|00000000",
+    "sourceGame": ["rct2"],
+    "objectType": "music",
+    "properties": {
+        "originalStyleId": 30,
+        "tracks": [
+            {
+                "source": "$RCT2:DATA/css44.dat",
+                "name": "Drunken Sailor",
```

Comment:
I was going to comment on this, but Wikipedia appears to favour "Drunken Sailor".